### PR TITLE
Release notes for 20.05

### PR DESCRIPTION
**Pull request by mvdbeek**
Please help!

### CODE REVIEW

File: doc/source/releases/20.05_announce_user.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+* Add tpr datatype
+  (thanks to `@simonbray <https://github.com/simonbray>`__).
+  `Pull Request 9244`_
+* Python3 compatible SnpEff and SnpSift metadata detection
+  (thanks to `@wm75 <https://github.com/wm75>`__).
+  `Pull Request 9282`_
+* Add datatypes for xcms and camera
+  (thanks to `@computational-metabolomics <https://github.com/computational-metabolomics>`__).
+  `Pull Request 9470`_
+* index with -c flag for large bam (one region larger than 2**29 - 1)
```
Comment (nsoranzo) [inline doc/source/releases/20.05_announce_user.rst]:
```suggestion
* index with ``-c`` flag for large bam (one region larger than ``2**29 - 1``)
```

File: doc/source/releases/20.05_announce_user.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+Highlights
+===========================================================
+
+**Mass-Spec datatypes**
+  @bernt-matthias, can you write a bit about this?
+
+**New Interactive tools**
+  @astrovsky01, can you write this ?
+
+**Feature3**
```
Comment (hexylena) [inline doc/source/releases/20.05_announce_user.rst]: I'd recommend the following highlights, maybe in this order:

1. Tool recommendations (even if not on every galaxy, can be, and it is a user requested/facing feature.)
2. New ITs
3. Fixed the rule builder scrolling (huge bug for many medium power users.), todo find xref.
3. OIDC Login "yay social auth, login with fav provider blah blah"
  - https://github.com/galaxyproject/galaxy/pull/9060
  - Connect/disconnect interface
  - https://github.com/galaxyproject/galaxy/pull/9162 logout
  - elixir aai saw improvements


ITs is good, mass-spec is maybe too specific for this announcement?

I'd recommend against startup speed + tool testing improvements, these are really end user announcements and should focus on things people who don't run or develop galaxy, people who just show up to .org/.eu

Comment (mvdbeek) [reply doc/source/releases/20.05_announce_user.rst]: Fair, I'll move / drop those.

File: doc/source/releases/20.05_announce_user.rst [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+.. figure:: images/20.05-tool-prediction.gif
+   :alt: Tool prediction shown in the workflow interface
+
+**New Interactive Tools**
+  Through the combined efforts of many UseGalaxy.eu admins and developers, as well as the European Galaxy community we have converted a significant number of interactive environments into the new interactive tool specification. Thanks to `@bgruening <https://github.com/bgruening>`__, `@hexylena <https://github.com/hexylena>`__, `@jenzopr <https://github.com/jenzopr>`__, `@yvanlebras <https://github.com/yvanlebras>`__, `@annefou <https://github.com/annefou>`__, `@abretaud <https://github.com/abretaud>`__, `@joachimwolff <https://github.com/joachimwolff>`__, and `@astrovsky01 <https://github.com/astrovsky01>`__. You can try them all on `live.usegalaxy.eu <https://live.usegalaxy.eu/>`__
+
+.. figure:: images/20.05-new-its.png
+   :alt: Tool prediction shown in the workflow interface
+
+**Rule Based Uploader: Scolling!**
```
Comment (jdavcs) [inline doc/source/releases/20.05_announce_user.rst]:
```suggestion
**Rule Based Uploader: Scrolling!**
```

File: doc/source/releases/20.05_announce.rst [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  Feature description.
+
+**Data Tables can now be backed by refgenie**
+  @blankenberg, can you write a line about this
+
+**Tool Shed is now Python3 ready**
+  @mvdbeek will fill this
+
+**Many new Interactive Tools**
+  @astrovsky01 can you write a bit abbout this ?
```
Comment (astrovsky01) [inline doc/source/releases/20.05_announce.rst:22]: Galaxy interactive tools allow a greater depth of analysis via access to GUI-based tools inside an instance. 10 new interactive tools join the standard base toolset on galaxy download. Several tools previously only available on the usegalaxy.eu server, are now able to be run on local instances via configurations in the job_conf.xml and galaxy.yml files, details of which can be found in the documentation under "Galaxy InteractiveTools". Improvements to the Higlass interactive tool also now allow for a more in-depth visualization of multiple datasets in various formats simultaneously (`Pull Request 9559`_).

File: doc/source/releases/20.05_announce_user.rst (hunk 1 above)
Comment (mvdbeek) [reply doc/source/releases/20.05_announce_user.rst]:
```suggestion
* Create csi index for bam files where any region is larger than ``2**29 - 1``)
```

File: doc/source/releases/20.05_announce.rst [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  Feature description.
+
+**Data Tables can now be backed by refgenie**
+  @blankenberg, can you write a line about this
+
+**Tool Shed is now Python3 ready**
+  @mvdbeek will fill this
+
+**Many new Interactive Tools**
+Galaxy interactive tools allow a greater depth of analysis via access to GUI-based tools inside an instance. 10 new interactive tools join the standard base toolset on galaxy download. Several tools previously only available on the usegalaxy.eu server, are now able to be run on local instances via configurations in the job_conf.xml and galaxy.yml files, details of which can be found in the documentation under "Galaxy InteractiveTools". Improvements to the Higlass interactive tool also now allow for a more in-depth visualization of multiple datasets in various formats simultaneously (`Pull Request 9559`_).
```
Comment (nsoranzo) [inline doc/source/releases/20.05_announce.rst]:
```suggestion
Galaxy interactive tools allow a greater depth of analysis via access to GUI-based tools inside an instance. 10 new interactive tools (some previously available only on the UseGalaxy.eu server) join the standard base toolset. Improvements to the Higlass interactive tool also now allow for a more in-depth visualization of multiple datasets in various formats simultaneously (`Pull Request 9559`_).
```

**Comment by mvdbeek** [comment]
I think this might be ready to go, please have a final look!

File: doc/source/releases/20.05_announce.rst [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+**Data Tables can now be backed by refgenie**
+`Refgenie <http://refgenie.databio.org/>`__ manages storage, access, and transfer of reference genome resources. Galaxy can now fill data tables that were received from a refgenie installation (`Pull Request 9600`_).
+
+**Tool Shed is now Python3 ready**
+As the last component in the Galaxy Codebase the Tool Shed has been ported to Python 3. This concludes a 4 year effort to port the Galaxy codebase to Python 3 (`Pull Request 9528`_).
+
+**Workflow Editor and Workflow Run Form in Vue.js**
+The Workflow Editor and Workflow Run Form have been re-written in Vue.js. While the functionality has been conserved this lays the groundwork for creating beatiful, and reactive custom variants of these components in the future (`Pull Request 9346`_, `Pull Request 9332`_, `Pull Request 9324`_, `Pull Request 9249`_, `Pull Request 9201`_, `Pull Request 9147`_, `Pull Request 9138`_, `Pull Request 9574`_).
```
Comment (nsoranzo) [inline doc/source/releases/20.05_announce.rst]:
```suggestion
The Workflow Editor and Workflow Run Form have been re-written in Vue.js. While the functionality has been conserved this lays the groundwork for creating beautiful and reactive custom variants of these components in the future (`Pull Request 9346`_, `Pull Request 9332`_, `Pull Request 9324`_, `Pull Request 9249`_, `Pull Request 9201`_, `Pull Request 9147`_, `Pull Request 9138`_, `Pull Request 9574`_).
```

**Pull request merged**